roles/openshift_logging/defaults/main.yml

```diff
@@ -27,7 +27,7 @@ openshift_logging_kibana_proxy_debug: false
 openshift_logging_kibana_proxy_cpu_limit: null
 openshift_logging_kibana_proxy_memory_limit: null
 openshift_logging_kibana_replica_count: 1
-openshift_logging_kibana_edge_term_policy: Redirect
+openshift_logging_kibana_edge_term_policy: null
```

Comment:
Shouldn't this be "None" to reflect the README? or should the README be updated?